Documentation/doc/resources/1.8.13/BaseDoxyfile.in

```diff
@@ -1960,7 +1962,7 @@ MAN_LINKS              = NO
 # captures the structure of the code including all documentation.
 # The default value is: NO.
 
-GENERATE_XML           = NO
+GENERATE_XML           = YES
```

Comment:
I don't think it should be ON by default. You can add an option in the cmake to switch it on/off.